src/transformState.js

```diff
@@ -47,6 +47,25 @@
  */
 function transformState(state, transforms) {
   // write code here
+  for (const trans of transforms) {
```

Comment:
```suggestion
  for (const transform of transforms) {
```
why did you shorten it?